force-app/main/default/triggers/AnotherOpportunityTrigger.trigger

```diff
@@ -19,94 +19,149 @@ https://www.salesforceben.com/12-salesforce-apex-best-practices/
 https://developer.salesforce.com/blogs/developer-relations/2015/01/apex-best-practices-15-apex-commandments
 */
 trigger AnotherOpportunityTrigger on Opportunity (before insert, after insert, before update, after update, before delete, after delete, after undelete) {
-    if (Trigger.isBefore){
-        if (Trigger.isInsert){
-            // Set default Type for new Opportunities
-            Opportunity opp = Trigger.new[0];
-            if (opp.Type == null){
-                opp.Type = 'New Customer';
-            }        
-        } else if (Trigger.isDelete){
-            // Prevent deletion of closed Opportunities
-            for (Opportunity oldOpp : Trigger.old){
-                if (oldOpp.IsClosed){
-                    oldOpp.addError('Cannot delete closed opportunity');
-                }
+    new OpportunityTriggerHandler().run();
```

Comment:
This can be commented out since you're doing the same thing on the Opportunity trigger